test.js

```diff
@@ -69,9 +69,7 @@ function dockerStart(imgName) {
         console.log('starting container from image: %s', imgName);
         docker.createContainer({
           Image: imgName,
-          HostConfig: {
-            PublishAllPorts: true,
-          },
+          Cmd: '-d -p 5984:5984 klaemo/couchdb:2.0-dev --with-haproxy --admin=admin:pass',
```

Comment:
For reference, I pulled these env vars directly from the startup script since they aren't described very clearly in the actual `README.md`: https://github.com/apache/couchdb-docker/blob/master/2.0.0/docker-entrypoint.sh